ypathgen/testdata/structs/openconfig-simple-31.path-txt

```diff
@@ -14,32 +14,180 @@ import (
 	"github.com/openconfig/ygot/ygot"
 )
 
```

Comment:
Is the diff here solely because of the change to the algorithm for splitting that you make above?

Reply:
Yes, due to ceiling being used on the structN/fileN to determine structs per file.

Reply:
Great, thanks.